### PR TITLE
BW-1342 Support for passing entity field and ID to CBAS

### PR DESCRIPTION
**Pull request by salonishah11**
Note to reviewers: `TextInput` in `input.js` is copied from Terra UI

Testing: Local against local CBAS with Chris' changes from [PR #33](https://github.com/DataBiosphere/cbas/pull/33)

Screenshot of what the new inputs page looks like. The placeholders are added to let user know what is the expected structure of input:

After filling up the fields:
Screen Shot 2022-08-17 at 11 47 02 AM

Successful response from CBAS after submitting the workflow:
Screen Shot 2022-08-17 at 11 47 59 AM

Closes https://broadworkbench.atlassian.net/browse/BW-1342

### CODE REVIEW

**Review by kpierre13 — APPROVED** [review]

File: src/pages/SubmitWorkflow.js [hunk 1]
```diff
@@ -41,7 +43,16 @@ export const SubmitWorkflow = () => {
 
   const submitRun = async () => {
     try {
-      await Ajax(signal).Cbas.runs.post(workflowUrl, workflowInputs)
+      const runSetsPayload = {
+        workflow_url: workflowUrl,
+        workflow_param_definitions: JSON.parse(workflowInputsDefinition),
+        wds_entities: {
+          entity_type: entityType,
+          entity_ids: JSON.parse(entityId)
```
Comment (cjllanwarne) [inline src/pages/SubmitWorkflow.js:51]: Curious why the entityId needs parsing first?

File: src/pages/WorkflowInputs.js [hunk 2]
```diff
@@ -1,28 +1,74 @@
...
-import { TextArea } from 'src/components/input'
+import { TextArea, TextInput } from 'src/components/input'
 import { FormLabel } from 'src/libs/form'
 
 
-export const WorkflowInputs = ({ workflowUrl, workflowInputs, setWorkflowInputs }) => {
+export const WorkflowInputs = ({ workflowUrl, entityType, setEntityType, entityId, setEntityId, workflowInputsDefinition, setWorkflowInputsDefinition }) => {
+  // used as placeholder to let users know expected structure of inputs definition.
+  // To be removed later when we design UI for entering the input definition.
+  const inputMappingExample = '\n[\n' +
```
Comment (cjllanwarne) [inline src/pages/WorkflowInputs.js:11]: Would it be easier to write this out in JSON and then use a `toString` method to generate the text box contents?

**Review by cjllanwarne — APPROVED** [review]
LGTM, with a few minor questions/suggestions

File: src/pages/SubmitWorkflow.js (hunk 1 above)
Comment (salonishah11) [reply src/pages/SubmitWorkflow.js:51]: Without the parsing, the array was passed as an escaped String and CBAS would return 400 Bad Request

File: src/pages/WorkflowInputs.js (hunk 2 above)
Comment (salonishah11) [reply src/pages/WorkflowInputs.js:11]: Good point. I will try that.

**Pull request merged**